project_scrum/models/project_task.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
+# Copyright <2017> <Tenovar Ltd>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models, fields, api
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+    _order = "sequence"
+
+    @api.model
+    def create(self, vals):
+        vals['reference'] = self.env['ir.sequence'].next_by_code(
+            'project.Task')
+        return super(ProjectTask, self).create(vals)
+
+    @api.model
+    def _get_moscow_field(self):
+        return [('1', 'must'),
+                ('2', 'should'),
+                ('3', 'could'),
+                ('4', 'wont'),
+                ('not_set', 'Not Set')]
+
+    @api.model
+    def _get_value_field(self):
+        return [
+            ('0', '0'),
+            ('1', '1'),
+            ('2', '2'),
+            ('3', '3'),
+            ('5', '5'),
+            ('8', '8'),
+            ('13', '13'),
+            ('20', '20'),
+            ('40', '40'),
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_risk_field(self):
+        return [
+            ('0', '0'),
+            ('1', '1'),
+            ('2', '2'),
+            ('3', '3'),
+            ('5', '5'),
+            ('8', '8'),
+            ('13', '13'),
+            ('20', '20'),
+            ('40', '40'),
+            ('100', '100'),
+            ('00', 'Not Set'),
+        ]
+
+    @api.model
+    def _get_kano_field(self):
+        return [
+            ('excitement', 'Excitement'),
+            ('indifferent', 'Indifferent'),
+            ('mandatory', 'Mandatory'),
+            ('performance', 'Performance'),
+            ('questionable', 'Questionable'),
+            ('reverse', 'Reverse'),
+            ('not_set', 'Not Set'),
+        ]
+
+    user_id = fields.Many2one(
```

Comment:
this and the date fields look like redefinitions from the project module. Remove if that's true